lib/google_search_data_viewer_worker/keywords/helpers/job_creation.ex

```diff
@@ -0,0 +1,14 @@
+defmodule GoogleSearchDataViewerWorker.Keywords.JobCreationHelper do
+  alias GoogleSearchDataViewerWorker.Keywords.SearchWorker
+
+  def create_keyword_upload_jobs_with_delay(keyword_uploads, delay \\ 3) do
+    keyword_uploads
+    |> Enum.zip_with(
+      Enum.map(Enum.to_list(0..(Enum.count(keyword_uploads) - 1)), &(&1 * delay)),
+      fn keyword_upload, delay ->
+        SearchWorker.new(%{keyword_id: keyword_upload.id}, schedule_in: delay)
+      end
+    )
```

Comment:
Correct me if I'm misunderstood 😄 This function is to build jobs with delay, right? I think we can use `Enum.with_index/2` to make the function easier to read.

BTW good idea to add the delay here 👍
```suggestion
    |> Enum.with_index()
    |> Enum.map(fn {keyword_upload, index} ->
        SearchWorker.new(%{keyword_id: keyword_upload.id}, schedule_in: index * delay)    
      end
    )
```

Reply:
Yes, it handles creating the jobs with a `three` second delay for each.

Nice, definitely improves readability.